src/installer/install.ts

```diff
@@ -121,6 +121,12 @@ export async function downloadAndInstall(file: ClientFile): Promise<string> {
 }
 
 export async function saveIntoCache(clientExecutablePath: string, file: ClientFile): Promise<void> {
+    if (isGhes()) {
+        ghCore.info(`Cache is not supported on GitHub Enterprise Server; skipping cache saving. `
```

Comment:
'GitHub enterprise detected; skipping cache upload. For more information, see (issue)'